src/components/common/motd-modal/motd-renderer.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useMemo } from 'react'
+import { GenericSyntaxMarkdownExtension } from '../../markdown-renderer/markdown-extension/generic-syntax-markdown-extension'
+import { useConvertMarkdownToReactDom } from '../../markdown-renderer/hooks/use-convert-markdown-to-react-dom'
+import { LinkifyFixMarkdownExtension } from '../../markdown-renderer/markdown-extension/linkify-fix-markdown-extension'
+import { EmojiMarkdownExtension } from '../../markdown-renderer/markdown-extension/emoji/emoji-markdown-extension'
+import { DebuggerMarkdownExtension } from '../../markdown-renderer/markdown-extension/debugger-markdown-extension'
+import { useApplicationState } from '../../../hooks/common/use-application-state'
+import { ProxyImageMarkdownExtension } from '../../markdown-renderer/markdown-extension/image/proxy-image-markdown-extension'
+import { YoutubeMarkdownExtension } from '../../markdown-renderer/markdown-extension/youtube/youtube-markdown-extension'
+import { AlertMarkdownExtension } from '../../markdown-renderer/markdown-extension/alert-markdown-extension'
+import { SpoilerMarkdownExtension } from '../../markdown-renderer/markdown-extension/spoiler-markdown-extension'
+import { BlockquoteExtraTagMarkdownExtension } from '../../markdown-renderer/markdown-extension/blockquote/blockquote-extra-tag-markdown-extension'
+import { VimeoMarkdownExtension } from '../../markdown-renderer/markdown-extension/vimeo/vimeo-markdown-extension'
+
+/**
+ * Reads the motd from the global application state and renders it as markdown with a subset of the usual features and without HTML support.
+ */
+export const MotdRenderer: React.FC = () => {
+  const extensions = useMemo(
+    () => [
+      new YoutubeMarkdownExtension(),
```

Comment:
YouTube and Vimeo don't make much sense in the motd, do they?

Reply:
Why?

Reply:
Because the motd is a banner at the top of the page? Doesn't seem like the best place for a video

Reply:
Which banner? We replaced it a long time ago with a modal.

Reply:
And it's not a banner anymore. Well then this is irrelevant

Reply:
This one:

<img width="388" alt="image" src="https://user-images.githubusercontent.com/6124140/153365477-c5f0ca12-95ff-4d20-9f87-31ed5d7db099.png">